qemu/tests/win_irq_check.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import re
+
+from virttest import error_context
+from virttest import utils_test
+
+
+# This decorator makes the test function aware of context strings
+@error_context.context_aware
+def run(test, params, env):
+    """
+    QEMU windows guest vitio device irq check test
+
+    1) Start guest with virtio device.
+    2) Make sure driver verifier enabled in guest.
+    3) Get irq info in guest.
+    4) Check the value of irq number.
+
+    :param test: QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+    def irq_check(session, device_name):
+        """
+        Return if virtio device's irq number
+        :param session: use for sending cmd
+        :param device_name: virtio device's name
+        :param driver_name: virtio driver's name
+        """
+        status, irq_dev_info = session.cmd_status_output(params["irq_cmd"]
+                                                         % device_name)
+        if status:
+            test.fail("Can't get %s's irq info." % device_name)
+        irq_value = re.split('\s+', irq_dev_info)[1]
+        logging.info("irq number is %s" % irq_value)
+        return int(irq_value)
+
+    driver = params["driver_name"]
+    device_name = params["device_name"]
+    timeout = int(params.get("login_timeout", 360))
+    error_context.context("Boot guest with %s device" % driver, logging.info)
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+
+    check_status, output = utils_test.qemu.check_win_driver(test, driver, vm, timeout)
+    if check_status:
```

Comment:
There is no return value in patch1201, so no need to "check_status, output", I think just use this "utils_test.qemu.check_win_driver(test, driver, vm, timeout)" is ok.

Reply:
Patch 1201 is updated.Pls have a check again.